src/__tests__/tab.js

```diff
@@ -244,6 +244,26 @@ test('should suport a mix of elements with/without tab index', () => {
   expect(radio).toHaveFocus()
 })
 
+test('ignore tabindex when active element has tabindex="-1"', () => {
+  setup(`
+    <input tabindex='1'/>
+    <input tabindex='0'/>
+    <input tabindex='-1'/>
+    <input tabindex='2'/>
+  `)
+  const [inputA, inputB, inputC, inputD] = document.body.lastChild.children
```

Comment:
Rather than using `document.body.lastChild`, could we get that element from `const {element} = setup('...html...')`?

Reply:
Noticed the recent push. Don't miss this one :)

Reply:
done.

While we're at it:
I think the extra `<div>` in most tests to comply with the implementation of `setup()` is no improvement.

What do you think about rewriting that `setup()` function as:
```js
function setup(ui, {eventHandlers} = {}) {
  const div = document.createElement('div')
  div.innerHTML = ui.trim()
  document.body.append(div)

  return {
    element: div.firstChild,
    elements: div.children,
    ...addListeners(div, {eventHandlers}),
  }
}
```

Reply:
Sure! Let's do that in another PR though. I'll merge this as-is 👍

Reply:
Looks like this line hasn't been updated yet.